src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,96 @@
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { MovieCard } from '../MovieCard';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+type Props = {
+  addMovie: (movie: Movie) => void
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [searchTitle, setSearchTitle] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [isMovieFind, setFindCheck] = useState(true);
+
+  const searchInput = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchTitle(event.target.value);
+  };
+
+  const movieFind = async () => {
+    try {
+      const movieFromServer = await getMovie(searchTitle);
+
+      setMovie(movieFromServer);
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+      if (movieFromServer.Response === 'False') {
+        setFindCheck(false);
+        setMovie(null);
+      }
+    } catch {
+      throw new Error();
+    }
+  };
+
+  const addMovieToList = (movieToList: Movie) => {
+    if (movieToList) {
+      addMovie(movieToList);
+      setMovie(null);
+    }
+
+    setSearchTitle('');
+  };
+
+  return (
+    <>
+      <form className="find-movie">
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              value={searchTitle}
+              onChange={searchInput}
+              className={`input ${!isMovieFind ? 'is-danger' : ''}`}
+            />
+          </label>
+
+          {!isMovieFind && (
+            <p className="help is-danger">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
-        <div className="control">
-          <button
-            type="button"
-            className="button is-primary"
-          >
-            Add to the list
-          </button>
+        <div className="field is-grouped">
+          <div className="control">
+            <button
+              type="button"
+              className="button is-light"
+              onClick={movieFind}
+            >
+              Find a movie
+            </button>
+          </div>
+
+          <div className="control">
+            <button
+              type="button"
+              className="button is-primary"
+              onClick={() => movie && addMovieToList(movie)}
```

Comment:
check inside addMovieToList

```suggestion
              onClick={() => addMovieToList(movie)}
```